src/utils/terminal_utils.rs

```diff
@@ -0,0 +1,67 @@
+use crossterm::{
+  event::{DisableMouseCapture, EnableMouseCapture},
+  execute,
+  terminal::{self, EnterAlternateScreen, LeaveAlternateScreen},
+};
+use std::io::{self, stdout};
+use tui::{
+  backend::CrosstermBackend,
+  style::{Color, Modifier, Style},
+  text::Span,
+  widgets::{Block, Borders},
+  Terminal,
+};
+
+pub type TerminalBackendTp = CrosstermBackend<io::Stdout>;
+pub type TerminalBackend<W> = CrosstermBackend<W>;
+
+pub fn create_terminal_backend() -> Result<TerminalBackendTp, io::Error> {
+  if let Ok(_) = terminal::enable_raw_mode() {
```

Comment:
Now that the return type is **Result<_,io::Error>** I think it's better to use the error propagation operator **?**, instead of nested ifs

Reply:
Sure 👍🏼